### DISCUSSION TITLE
Two-hand touch support for desktop platforms

### DISCUSSION THREAD

**Discussion by cihe13375** [ppy/osu#23243] [Q&A] [answered]
It seems that two-hand touch support has been introduced a while ago. If I understand it correctly, it allows tapping in empty spaces in the playfield with one hand, while following the objects with the other hand. I tried this on Android and it works as expected. However, it does not work on my Surface Pro, even with the "Touch" option enabled. Is two-hand touch input on desktop platforms supported now?
If not, will it be supported in the future?

**Comment by Theighlin** [comment, marked as answer]
Do you have mouse buttons disabled? I don't know why that would make a difference but it seemed to me that it doesn't work on android with the setting off.

**Reply by cihe13375** [reply]
I tried it again today and it's working now, I don't know why. Probably because I pressed down the aim cursor before the map begins in my tests yesterday (in that case it doesn't work). And yes, it requires the mouse buttons enabled.

**Comment by cihe13375** [comment]
Another thing I noticed: if I use Surface Pen as the aim cursor, the pen may stop function at random times during play (this is more likely to occur in denser maps). This also happens when playing Relax (using pen as sole input), and I have to restart the computer to make the pen function again (ouch). It's more likely a driver issue though, I guess.
